config/activity_labels/recipe_coffee.yaml

```diff
@@ -8,53 +9,97 @@ labels:
     label: "background"
   - id: 1
     label: "measure-12oz-water"
+    full_str: "Measure 12 ounces of water in the liquid measuring cup"
+    depends: ["start"]
   - id: 2
     label: "pour-water-kettle"
+    full_str: "Pour the water from the liquid measuring cup into the electric kettle"
+    depends: ["measure-12oz-water"]
   - id: 3
-    label: "turn-kettle-on"
-  - id: 4
     label: "place-dipper-on-mug"
-  - id: 5
+    full_str: "Place the dripper on top of a coffee mug"
+    depends: ["start"]
+  - id: 4
     label: "filter-fold-half"
-  - id: 6
+    full_str: Take the coffee filter and fold it in half to create a semi-circle
+    depends: ["start"]
+  - id: 5
     label: "filter-fold-quarter"
-  - id: 7
+    full_str: "Fold the filter in half again to create a quarter-circle"
+    depends: ["filter-fold-half"]
+  - id: 6
     label: "place-filter"
-  - id: 8
+    full_str: "Place the folded filter into the dripper such that the the point of the quarter-circle rests in the center of the dripper"
+    depends: ["filter-fold-quarter"]
+  - id: 7
     label: "spread-filter"
-  - id: 9
+    full_str: "Spread the filter open to create a cone inside the dripper"
+    depends: ["place-filter"]
+  - id: 8
     label: "scale-turn-on"
-  - id: 10
+    full_str: "Turn on the kitchen scale"
+    depends: ["spread-filter"]
```

Comment:
This can happen independently from filter actions. "start" may be the appropriate dependency here.

Reply:
I put it here because the scale tends to turn itself off after a bit. So if we turn on the scale too soon, it'll just turn off and we need to repeat the action again. Not sure if we are okay with repeated turning on actions (this also applies to the thermometer)